packages/create-skill-app/templates/next/.config-templates/env.development

```diff
@@ -16,3 +16,6 @@ NEXT_PUBLIC_GOOGLE_ANALYTICS=CHANGE_ME
 # ConvertKit Newsletter Form
 NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM=FORM_ID
 NEXT_PUBLIC_CONVERTKIT_TOKEN=CK_TOKEN
+NEXT_PUBLIC_CONVERTKIT_ANSWER_URL=/api/skill/answer/convertkit
+NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL=/api/skill/subscribe/convertkit
+NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL=/api/skill/subscriber/convertkit
```

Comment:
template is configured appropriately for new endpoints!

Reply:
nice!